nevergrad/functions/ml/mlfunctionlib.py

```diff
@@ -79,16 +79,21 @@ def _ml_parametrization(
         if noise_free:  # noise_free is True when we want the result on the test set.
             X_test = self.X_test
             y_test = self.y_test
-            regr.fit(self.X_train, self.y_train)
+            if regressor == "kerasDenseNN":
```

Comment:
can you update the docstring to include this? and rename it to fit the same style as other names, so probably keras_dense_nn.

Also, inistead of having two similar if/else, please use something like to factorize:
```python
fit_additional_params = dict(verbose=0, epochs=350) if regressor == "keras_dense_nn" else {}
```
then just `regr.fit(X, y, **fit_additional_params)` instead of the if condition. This shortens the code and would prevent forgetting to change in one place if we change in the others

Reply:
Right, it's better - done.